.github/workflows/ci-build.yaml

```diff
@@ -71,6 +71,9 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - build-go
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
GITHUB_TOKEN is a reserved secret name in Github. Please use E2E_TEST_GITHUB_TOKEN and E2E_TEST_GITLAB_TOKEN instead.

Reply:
thnx! done! , i hope u meant for secrets name and not env variable

Reply:
@alexmt : please re review! eagerly waiting to see this fix, post merge test runs